### PR TITLE
JR2-1263 Update phone validation regex

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiApplyChannelDto.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
     private String emailAddress;
 
     @Size(max=20)
-    @Pattern(regexp = "[+][0-9]{11,}")
+    @Pattern(regexp = "[+][0-9]{10,}")
```
Comment (birom) [inline application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiApplyChannelDto.java]: I'd suggest to create a ``@PhoneNumber`` annotation like the [CountryIsoCode](https://github.com/alv-ch/job-ad-service/blob/master/domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/utils/CountryIsoCode.java).

Comment (varusakov) [reply application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiApplyChannelDto.java]: Good point, tnx

File: application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiContactDto.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
     @NotBlank
     @Size(max=20)
```
Comment (birom) [inline application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiContactDto.java]: Can't we move the ```@Size``` also into the ```@PhoneNumber```?

Comment (varusakov) [reply application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiContactDto.java]: I think we could change regex to `[+][0-9]{10,19}` and remove `@Size`

Comment (birom) [reply application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/jobadvertisement/dto/api/ApiContactDto.java]: Yes , of course it also works. But in this case please write a test to make sure we never allow logner strings.

**Pull request merged**